apertium/apertium_re.cc

```diff
@@ -44,9 +32,8 @@ re(0)
 
 ApertiumRE::~ApertiumRE()
 {
-  if(!empty)
-  {
-    pcre_free(re);
+  if(!empty) {
```

Comment:
Can this state simply be whether the pointer is nullptr or not? Do we need `empty`?